http_action.go

```diff
@@ -2,13 +2,12 @@ package gotfy
 
 import (
 	"encoding/json"
-	"net/url"
 )
 
 // HttpAction allows attaching an HTTP request action to a notification
 type HttpAction[X comparable] struct {
 	Label   string            `json:"label"`   //  string  -  Open garage door  Label of the action button in the notification
-	URL     *url.URL          `json:"url"`     //  string  -  https://ntfy.sh/mytopic  URL to which the HTTP request will be sent
+	URL     string            `json:"url"`     //  string  -  https://ntfy.sh/mytopic  URL to which the HTTP request will be sent
```

Comment:
So I think `omitempty` might be good here. Just to shrink payload
```suggestion
	URL     string            `json:"url,omitempty"`     //  string  -  https://ntfy.sh/mytopic  URL to which the HTTP request will be sent
```